TYPO3.TYPO3CR/Resources/Private/Schema/NodeTypes.schema.yaml

```diff
@@ -65,6 +65,16 @@ additionalProperties:
             # we intentionally do not specify which properties are allowed here;
             # as other packages (such as TYPO3.Neos) might do that instead.
 
+          'options':
+            type: dictionary
+            # we intentionally do not specify which properties are allowed here;
+            # as other packages might do that instead.
+
+          'search':
```

Comment:
this doesn't belong in the core as it's not used by the core.. instead it should be added in the `content-repository-search` package